doc/quickstart.md

```diff
@@ -65,7 +65,7 @@ On CentOS and openSuSE:
     yum install -y yum-plugin-copr epel-release
     yum copr -y enable jhli/ipmctl
     yum copr -y enable jhli/safeclib
-    yum install -y libipmctl-devel
+    yum install -y libipmctl-devel python-devel
```

Comment:
No.  Don't add this here.  It was fine where it was before.  Or add it to one of the `yum install` commands above line 64.

Reply:
ok.